src/SB/Game/zNPCFXCinematic.cpp

```diff
@@ -1,13 +1,18 @@
 #include "xVec3.h"
 #include "xMath3.h"
+#include "xParEmitter.h"
 
 #include "zNPCFXCinematic.h"
 
 #include <types.h>
-#include "xCutscene.h"
 
 static S32 init = 0;
 
+static const F32 zero = 0.0f;            
+static const F32 addVal = 20.0f;         
+static const F32 fillVec3Val = 5.0f;
```

Comment:
These symbols do not exist in the symbol map contained within objdiff. You'll need to get rid of them and replace them with their literal values where used to increase the matchingness.

Reply:
References for this statement:
![image](https://github.com/user-attachments/assets/cf928172-a33b-4e77-91c9-f8ed96cffb11)
![image](https://github.com/user-attachments/assets/06549c7b-a9da-4ace-b9c9-1469f34852db)